Framework/src/QualityControlFactory.cxx

```diff
@@ -0,0 +1,104 @@
+///
+/// \file   QualityControlFactory.cxx
+/// \author Piotr Konopka
+///
+
+#include <Configuration/ConfigurationFactory.h>
+#include "QualityControl/QualityControlFactory.h"
+#include "QualityControl/TaskRunnerFactory.h"
+#include "QualityControl/CheckerFactory.h"
+#include <boost/property_tree/ptree.hpp>
+#include <QualityControl/HistoMerger.h>
+#include <QualityControl/TaskRunner.h>
+
+using namespace o2::framework;
+using namespace o2::configuration;
+using namespace o2::quality_control::checker;
+using boost::property_tree::ptree;
+
+namespace o2
+{
+namespace quality_control
+{
+namespace core
+{
+
+QualityControlFactory::QualityControlFactory()
```

Comment:
you could use "=default"

Reply:
Since the methods are static anyway, then how about deleting constructors?

Reply:
yes, you are right.